deployment/overlays/balloons/kustomization.yaml

```diff
@@ -5,8 +5,8 @@ namespace: kube-system
 
```

Comment:
Minor nit about the commit message. Please change the commit subject to
`config: Fix configuration files for kustomize`
and add also signed-off-by as we use that in this project.